src/components/Tag.js

```diff
@@ -110,43 +129,72 @@ const Tag = ({ color, disableMenu, slug, text, title, maxValue, ...props }) => {
               '?search[stringTraits][0][name]=' +
               title +
               '&search[stringTraits][0][values][0]=' +
-              text
+              originalValue ?? text
           );
+        } else if (idx === 1) {
+          const url = originalValue ?? text;
```

Comment:
```suggestion
          const url = originalValue;
```
I don;'t get it. If this is a URL, then we should use `originalValue`, no?

Reply:
Just like I mentioned above I wanted to have a way to make `originalValue` optional